.travis.yml

```diff
@@ -17,6 +17,8 @@ gemfile:
 
 matrix:
   exclude:
+    - rvm: 2.2.3
+      gemfile: Gemfile.rails5
```

Comment:
It sounds like you did not fetch the latest code from this repository. Try this:

`git remote -v` to identify the right remote, for example on my machine I have origin pointing to my personal account and upstream to the official ActiveMerchant one:
```
origin	git@github.com:bdewater/offsite_payments.git (fetch)
origin	git@github.com:bdewater/offsite_payments.git (push)
upstream	git@github.com:activemerchant/offsite_payments.git (fetch)
upstream	git@github.com:activemerchant/offsite_payments.git (push)
```
If you don't have a remote pointing to the activemerchant/offsite_payments repository, add it with `git remote add upstream git@github.com:activemerchant/offsite_payments.git`

`git pull --rebase remote master`, where I would need to replace `remote` with `upstream`. Fix the merge conflicts and make sure at the end there are no changes in the Travis file. Then `git push --force` your branch to your personal fork and this pull request will be updated. Then the CI should run again and point out if all the tests are passing.